apps/web/src/server/applications/case/representations/representation/check-answers/utils/common-validators.js

```diff
@@ -0,0 +1,38 @@
+import { createValidator } from '@pins/express';
+import { body } from 'express-validator';
+
+export const validateRepresentedAddress = createValidator(
```

Comment:
what is common about these ?

All the keys are specific to this page. does not seem necessary lifting this here and not just doing it in the validator itself

Reply:
Good point, the initial idea was to use the same validators as the other pages but due to the keys being specific to this page I should move these out and into the .validation file.